API/Resources/AutocompleteProjectsResource.cs

```diff
@@ -0,0 +1,18 @@
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+
+namespace API.Resources
+{
+    /// <summary>
+    ///     The viewmodel that is returned by the Autocomplete search route in the Project controller.
+    /// </summary>
+    public class AutocompleteProjectsResource
+    {
+        /// <summary>
+        ///     The List of Autocompleted search projects.
+        /// </summary>
+        public List<AutocompleteProjectResource> AutocompleteProjects { get; set; }
+    }
+}
```

Comment:
I don't really understand the purpose of this class. Would you be able to just use the List<AutocompleteProjectResource>? I may be overlooking something though.

Reply:
I agree, I removed this class as we also don't use it for other Lists.